test/integration/anchorplatform.test.ts

```diff
@@ -0,0 +1,144 @@
+import { Wallet } from "../../src";
+import { IssuedAssetId } from "../../src/walletSdk/Asset";
+
+let wallet;
+let stellar;
+let anchor;
+let accountKp;
+const anchorUrl = "http://localhost:8080";
+
+describe("Anchor Platform Integration Tests", () => {
+  beforeAll(async () => {
+    // Setup
+    wallet = Wallet.TestNet();
+    stellar = wallet.stellar();
+    anchor = wallet.anchor({ homeDomain: anchorUrl, allowHttp: true });
+    accountKp = stellar.account().createKeypair();
+    await stellar.fundTestnetAccount(accountKp.publicKey);
+  }, 15000);
+
+  it("SEP-10 auth should work", async () => {
+    const auth = await anchor.sep10();
+    const authToken = await auth.authenticate({ accountKp });
+    expect(authToken.token).toBeTruthy();
+  });
+
+  it("SEP-12 KYC and SEP-6 should work", async () => {
+    const auth = await anchor.sep10();
+    const authToken = await auth.authenticate({ accountKp });
+
+    // add USDC trustline
+    const asset = new IssuedAssetId(
+      "USDC",
+      // anchor platform USDC issuer
+      "GDQOE23CFSUMSVQK4Y5JHPPYK73VYCNHZHA7ENKCV37P6SUEO6XQBKPP",
+    );
+    const txBuilder = await stellar.transaction({
+      sourceAddress: accountKp,
+    });
+    const addUsdcTx = txBuilder.addAssetSupport(asset).build();
+    addUsdcTx.sign(accountKp.keypair);
+    await stellar.submitTransaction(addUsdcTx);
+
+    // add SEP-12 KYC info
+    const sep12 = await anchor.sep12(authToken);
+    const sep12Resp = await sep12.add({
+      sep9Info: {
+        first_name: "john",
+        last_name: "smith",
+        email_address: "123@gmail.com",
+        bank_number: "12345",
+        bank_account_number: "12345",
+      },
+    });
+    expect(sep12Resp.data.id).toBeTruthy();
+
+    // SEP-6 deposit
+    const sep6 = anchor.sep6();
+    const dResp = await sep6.deposit({
+      authToken,
+      params: {
+        asset_code: "USDC",
+        account: accountKp.publicKey,
+      },
+    });
+    expect(dResp.id).toBeTruthy();
+
+    // SEP-6 withdraw
+    const wResp = await sep6.withdraw({
+      authToken,
+      params: {
+        asset_code: "USDC",
+        account: accountKp.publicKey,
+        type: "bank_account",
+        dest: "123",
+        dest_extra: "12345",
+      },
+    });
+    expect(wResp.id).toBeTruthy();
+  }, 30000);
+
+  it("SEP-24 should work", async () => {
+    const assetCode = "USDC";
+    const auth = await anchor.sep10();
+    const authToken = await auth.authenticate({ accountKp });
+
+    const dResp = await anchor.sep24().deposit({
+      assetCode,
+      authToken,
+    });
+    const transactionId = dResp.id;
+    expect(transactionId).toBeTruthy();
+
+    const wResp = await anchor.sep24().withdraw({
+      withdrawalAccount: accountKp.publicKey,
+      assetCode,
+      authToken,
+    });
+    expect(wResp.id).toBeTruthy();
+
+    const transaction = await anchor.sep24().getTransactionBy({
```

Comment:
Can you add a check for status ? 
Also, for withdrawal we can get transaction to pending_user_transfer_start

Reply:
FYI: current Kotlin implementation automatically moves transaction between statuses, so I think we should include watcher in the test case

Reply:
https://github.com/stellar/typescript-wallet-sdk/pull/99/commits/6251819e1cfe6d2582c3acb946ba8ca7d51c5258 👍 

for more full flow testing we already have tests in other files. If we want to do full anchor platform testing I think we can do when we move tests over, wdyt?

Reply:
Sounds good!